lib/reactTags.js

```diff
@@ -168,6 +167,12 @@ var ReactTags = React.createClass({
             });
         }
     },
+    handlePaste: function (e) {
+      e.preventDefault()
+      const clipboardData = e.clipboardData || window.clipboardData
+      const string = clipboardData.getData('text')
+      string.split(/[ ,]+/).forEach((buzzword) => this.props.handleAddition(buzzword))
```

Comment:
Why "buzzword" here? This should probably be "tag" to stay in-keeping with the libraries terminology